pkg/template/servicebroker/catalog.go

```diff
@@ -19,6 +19,8 @@ const (
 	// identity information.
 	requesterUsernameTitle       = "Template service broker: requester username"
 	requesterUsernameDescription = "OpenShift user requesting provision/bind"
+
+	noDescriptionProvided = "No description provided"
```

Comment:
Service.Description is a string field (not a pointer) so i assume we're sending effectively an emptystring today.